crates/rust-analyzer/src/config.rs

```diff
@@ -359,6 +359,8 @@ config_data! {
         imports_preferPrelude: bool                       = false,
         /// The path structure for newly inserted paths to use.
         imports_prefix: ImportPrefixDef               = ImportPrefixDef::Plain,
+        /// Whether to prefix external (including std, core) crate imports with `::`. e.g. "use ::std::io::Read;".
+        imports_prefix_external_crate: bool = false,
```

Comment:
This name collides with the above (one being the `prefix` object, the other being a field in said object, we should probably sanity check that those configs don't exist in a test) which will cause issues, `imports_prefixExternalPrelude` would be better

Reply:
Thanks for taking a look at my PR! Renamed it.